src/plugin/parse-metatags/src/java/org/apache/nutch/parse/metatags/MetaTagsParser.java

```diff
@@ -16,21 +16,22 @@
  */
 package org.apache.nutch.parse.metatags;
 
+import java.lang.invoke.MethodHandles;
 import java.util.Enumeration;
 import java.util.HashSet;
 import java.util.Locale;
 import java.util.Properties;
 import java.util.Set;
 
-import org.apache.commons.logging.Log;
```

Comment:
I wonder why this is even on the cp. Can we remove it?

Reply:
Classpath

Reply:
It's a transitive dependency of commons-jexl (and also of few libs required for tests). There are still a few Nutch classes using it instead of org.slf4j. I've opened [NUTCH-2747](https://issues.apache.org/jira/browse/NUTCH-2747) to address this.